tests/app/app.go

```diff
@@ -132,7 +129,6 @@ type TestApp struct {
 	MintKeeper            mintkeeper.Keeper
 	DistrKeeper           distrkeeper.Keeper
 	GovKeeper             *govkeeper.Keeper
-	CrisisKeeper          *crisiskeeper.Keeper
```

Comment:
q: Why remove?

Reply:
introduces a vulnerability that `govulncheck` complains about.  Not needed anyways since its just in the test app